app/controllers/businesses_controller.rb

```diff
@@ -10,12 +10,15 @@ def create
     @business = current_user.build_business
     @business.assign_attributes(permitted_attributes(@business))
 
-    if @business.save
-      url = stored_location_for(:user) || developers_path
-      event = Analytics::Event.added_business_profile(url)
-      redirect_to event, notice: t(".created")
-    else
-      render :new, status: :unprocessable_entity
+    respond_to do |format|
+      if @business.save
+        url = stored_location_for(:user) || developers_path
+        Analytics::Event.added_business_profile(url)
+        format.html { redirect_to stripe_checkout_path }
+        format.turbo_stream
```

Comment:
We still need to redirect to the analytics event so the tracking can take place.